forch/authenticator.py

```diff
@@ -24,10 +24,12 @@
 
 class Authenticator:
     """Authenticate devices using MAB/dot1x"""
-    def __init__(self, auth_config, auth_callback=None, radius_query_object=None):
+    def __init__(self, auth_config, auth_callback=None,
+                 radius_query_object=None, forch_metrics=None):
         self.radius_query = None
         self.sessions = {}
         self.auth_callback = auth_callback
+        self._forch_metrics = forch_metrics
```

Comment:
nit: at this point, it's really just "metrics" since, well, from this class' perspective, it doesn't know that it's forch or not.  Not a big deal (just a name), but something to ponder.

Reply:
Changed name of vars.